### ISSUE TITLE
Can I change isMobile property?

### ISSUE THREAD

**Issue by panay** [MikaelEdebro/vue-airbnb-style-datepicker#46]
I want to change the "isMobile" property because I have large mobile devices and in these, the datepicker looks wrong.

**Comment by MikaelEdebro** [comment]
Yeah this would be kinda useful. Would you prefer that this is a global config (when bootstrapping the plugin), or per datepicker basis?
Would it be suffice to have a prop/setting like:
`isMobileBreakpoint: 650`

**Comment by daamsie** [comment]
Might I suggest just "mobileBreakpoint"

The prefix "is" in "isMobileBreakpoint" implies to me that the datatype would be a boolean.
